$MirOS: ports/x11/icewm/patches/patch-lib_menu_in,v 1.5 2006/12/16 22:42:03 bsiegert Exp $
$OpenBSD: patch-lib_menu_in,v 1.2 2004/05/21 17:00:09 naddy Exp $
--- lib/menu.in.orig	Sun Sep 10 16:11:48 2006
+++ lib/menu.in	Tue Dec 26 15:24:24 2006
@@ -4,16 +4,128 @@
 # since modifications to this file will be discarded when you
 # (re)install icewm.
 #
-prog xterm xterm xterm
+# $MirOS: ports/x11/icewm/patches/patch-lib_menu_in,v 1.5 2006/12/16 22:42:03 bsiegert Exp $
+#
+prog UXTerm xterm uxterm
+prog XTerm xterm xterm
+prog Eterm Eterm Eterm
+prog aterm aterm aterm
 prog rxvt xterm rxvt -bg black -cr green -fg white -C -fn 9x15 -sl 500
-prog fte fte fte
-prog NEdit nedit nedit
-prog Mozilla mozilla mozilla
-prog XChat xchat xchat
-prog Gimp gimp gimp
 separator
-menuprog Gnome folder icewm-menu-gnome1 --list @CONFIG_GNOME1_MENU_DIR@
-menuprog Gnome folder icewm-menu-gnome2 --list @CONFIG_GNOME2_MENU_DIR@
-menuprog KDE folder icewm-menu-gnome@GNOME_VER@ --list @CONFIG_KDE_MENU_DIR@
-menufile Programs folder programs
-menufile Tool_bar folder toolbar
+#menufile Programs folder programs
+#menufile Tool_bar folder toolbar
+menu Applications folder {
+    menu Editors folder {
+        prog Jupp emacs uxterm -T Jupp -e jupp
+        prog ed-utf vim uxterm -T 'ed (UTF-8)' -e ed
+        prog ed-latin1 vim xterm -T 'ed (ISO-8859-1)' -e ed
+        prog NEdit nedit nedit
+        prog fte fte fte
+        prog vim vim gvim
+        prog xemacs xemacs xemacs
+        prog emacs emacs emacs
+        prog xed_it xedit xedit
+        prog Lyx emacs lyx
+        prog Anj_uta anjuta anjuta
+        prog Abiword abiword abiword
+        prog Xc_oral xcoral xcoral
+    }
+    menu "WWW Browsers" folder {
+        prog Lynx lynx uxterm -T Lynx -e lynx -display_charset utf8
+        prog Links+ lynx links -g
+        prog Netscape netscape netscape
+        prog Konqueror konqueror konqueror
+        prog Mozilla mozilla mozilla
+        prog Firefox firefox firesomething
+        prog Galeon galeon galeon
+        prog Opera opera opera
+        prog Dillo dillo dillo
+        prog Amaya amaya amaya
+    }
+    menu "Mail and News" folder {
+        prog Kmail kmail kmail
+        prog Sylpheed sylpheed sylpheed
+        prog Tk_Rat tkrat tkrat
+        prog Thunderbird thunderbird thunderbird
+	prog Elm elm xterm -T Elm -e elm
+	prog Mutt mutt xterm -T Mutt -e mutt
+	prog Pine pine xterm -T Pine -e pine
+    }
+    menu "Internet Utilities" folder {
+        prog sirc xchat uxterm -T sirc -e sirc -X
+	prog irssi xchat uxterm -T irssi -e irssi
+        prog XChat xchat xchat
+        prog GTK-Gnutella gtk-gnutella gtk-gnutella
+    }
+    menu Graphics folder {
+        prog Dia dia dia
+        prog Blender blender blender
+        prog gtksee gtksee gtksee
+        prog ImageMagic display display
+        prog PornView pornview pornview
+        prog Gimp gimp gimp
+        prog XV xv xv
+        prog XPaint xpaint xpaint
+        prog XFig xfig xfig
+    }
+    menu Multimedia folder {
+        prog XPlay_CD xplaycd xplaycd
+        prog XM_ixer xmixer xmixer
+        prog XMMS xmms xmms
+	prog "Audacious Media Player" audacious audacious
+        prog Mplayer gmplayer gmplayer
+    }
+    menu "System Utilities" folder {
+        prog xosview xosview xosview
+        prog HexCalc hexcalc hexcalc
+        prog "Northern Captain" xnc xnc
+        prog xfm xfm xfm
+        prog "Acrobat Reader" pdf acroread
+        prog Xpdf pdf xpdf
+        prog "DVI Previewer" xdvi xdvi
+        prog Ghostview ghostview gv
+	prog "Midnight Commander" mc uxterm -T mc -e mc -c
+    }
+}
+menu Games folder {
+    prog "Koules for X" koules xkoules -f
+    prog Xboing xboing xboing
+    prog Xboard xboard xboard
+    prog XGalaga xgalaga xgal
+    prog XDemineur xdemineur xdemineur
+    prog "X Scavenger" scavenger scavenger
+    prog Xkobo xkobo xkobo -doublesize
+    prog "Eboard (chess)" eboard eboard
+    prog Quake-2 quake2 quake2
+    prog Frozen-Bubble frozen-bubble frozen-bubble --colour-blind
+}
+menu Utilities folder {
+    prog top top xterm -e top
+    prog xload xload xload
+    prog Xman xman xman
+    prog "Font Selector" xfontsel xfontsel
+    prog Clock xclock xclock
+    prog Magnify xmag xmag
+    prog Calculator xcalc xcalc
+    prog Colormap xcolormap xcmap
+    prog Clip_board xclip xclipboard
+    prog xkill bomb xkill
+    separator
+    prog "Screen Saver" xlock xlock -nolock -mode random
+    prog "Screen _Lock" xlock xlock -mode random
+}
+menu "Window _Managers" folder {
+    restart icewm - icewm
+    restart evilwm - evilwm
+    restart wmaker - wmaker
+    restart enlightenment - enlightenment
+    restart blackbox - blackbox
+    restart fl_uxbox - fluxbox
+    restart fvwm_2 - fvwm2
+    restart fvwm - fvwm
+    restart w_m2 - wm2
+    restart pwm - pwm
+    restart twm - twm
+    restart KDE - startkde
+    restart Gnome - gnome-session
+}
